core/src/main/java/bisq/core/arbitration/DisputeManager.java

```diff
@@ -266,6 +268,13 @@ private boolean isReadyForTxBroadcast() {
     }
 
     private void applyMessages() {
+        // Some messages can't be handled until all the services are properly initialized.
+        // In particular it's not possible to complete the signing of a dispute payout
+        // by an encrypted wallet until after it's been decrypted.
+        if (!servicesInitialized) {
+            return;
```

Comment:
Just wondering if it would make more sense to put this check in tryApplyMessages? Since it appears to be the only caller of applyMessages and has an additional check.

Reply:
I moved to the readiness check, looks much more reasonable, thanks.

Reply:
Please ack again if it looks good.